tests/cupy_tests/core_tests/test_ndarray_reduction.py

```diff
@@ -20,6 +20,15 @@ def test_max_axis_large(self, xp, dtype):
         a = testing.shaped_random((3, 1000), xp, dtype)
         return a.max(axis=0)
 
+    """
+    @testing.attr.slow
+    @testing.for_dtypes(['?', 'b', 'h', 'e'])
+    @testing.numpy_cupy_allclose()
+    def test_max_axis_huge(self, xp, dtype):
+        a = testing.shaped_random((2048, 1024, 1024), xp, dtype)
+        return a.max(axis=0)
+    """
+
```

Comment:
Why is this test commented out? Also, I think the size of array should be greater than 2^31 to test huge integers.

(applies to other tests as well)